### PR TITLE
DOC : update documentation with the new carpetplot

### LINKED ISSUE #1039: Documentation of updated carpetplots
Dear all,

thank you for providing such a great tool. I think, however, that the documentation should be updated:

The documentation for the newer versions of MRIQC still uses the older carpetplot and neither explains the legend nor reference the meaning of the separations. Only after an extended search, I was able to find an answer in #1003. 

Best
Simon

### PR DESCRIPTION
**Pull request by celprov**
Update the carpetplot image in the documentation and add description of this new carpetplot.

These changes need to be reflected in the documentation of version 22.0.1 and onwards, but I don't know how to change the configuration of read the docs.

Closes #1039

### CODE REVIEW

File: docs/source/reports/bold.rst [hunk 1]
```diff
@@ -41,7 +41,12 @@ The section with visual reports contains:
    signal intensity at the extremes for the identification
    of spikes, the outliers metric, the DVARS and the
    :abbr:`FD (framewise displacement)`. Finally the
-   so-called carpetplot [Power2016]_
+   so-called carpetplot [Power2016]_. 
+   The carpet plot rows, corresponding to voxel timeseries,
+   are separated into regions : cortical gray-matter, deep
```
Comment (oesteban) [inline docs/source/reports/bold.rst:46]:
```suggestion
   and are separated into regions: cortical gray-matter, deep 
```

File: docs/source/reports/bold.rst [hunk 2]
```diff
@@ -41,7 +41,12 @@ The section with visual reports contains:
    signal intensity at the extremes for the identification
    of spikes, the outliers metric, the DVARS and the
    :abbr:`FD (framewise displacement)`. Finally the
-   so-called carpetplot [Power2016]_
+   so-called carpetplot [Power2016]_. 
+   The carpet plot rows, corresponding to voxel timeseries,
```
Comment (oesteban) [inline docs/source/reports/bold.rst:45]:
```suggestion
   The carpet plot rows correspond to voxelwise time series,
```

File: docs/source/reports/bold.rst [hunk 3]
```diff
@@ -41,7 +41,12 @@ The section with visual reports contains:
...
+   are separated into regions : cortical gray-matter, deep 
+   gray-matter, white-matter and cerebrospinal fluid, cerebellum
```
Comment (oesteban) [inline docs/source/reports/bold.rst:47]:
```suggestion
   gray-matter, white matter and cerebrospinal fluid, cerebellum 
```

File: docs/source/reports/bold.rst [hunk 4]
```diff
@@ -41,7 +41,12 @@ The section with visual reports contains:
    signal intensity at the extremes for the identification
    of spikes, the outliers metric, the DVARS and the
    :abbr:`FD (framewise displacement)`. Finally the
-   so-called carpetplot [Power2016]_
+   so-called carpetplot [Power2016]_. 
+   The carpet plot rows, corresponding to voxel timeseries,
+   are separated into regions : cortical gray-matter, deep 
+   gray-matter, white-matter and cerebrospinal fluid, cerebellum 
+   and crown [Provins2022]_. The crown corresponds to the voxels located on a
```
Comment (oesteban) [inline docs/source/reports/bold.rst:48]:
```suggestion
   and the brain-edge or “crown” [Provins2022]_.
   The crown corresponds to the voxels located on a 
```

File: docs/source/reports/bold.rst [hunk 5]
```diff
@@ -87,6 +92,15 @@ reported here.
 
 .. topic:: References
 
+  .. [Patriat2015] Patriat, R., EK Molloy, RM Birn, T. Guitchev, and A. Popov. “Using Edge Voxel Information to 
+     Improve Motion Regression for Rs-FMRI Connectivity Studies.” Brain Connectivity 5, no. 9 (28 2015): 582–95. 
+     doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`_.
+
   .. [Power2016] Power JD, A simple but useful way to assess fMRI scan qualities.
      NeuroImage. 2016. doi: `10.1016/j.neuroimage.2016.08.009 <http://doi.org/10.1016/j.neuroimage.2016.08.009>`_
```
Comment (oesteban) [inline docs/source/reports/bold.rst:100]:
```suggestion
     NeuroImage. 2016. doi: `10.1016/j.neuroimage.2016.08.009 <http://doi.org/10.1016/j.neuroimage.2016.08.009>`__.
```

File: docs/source/reports/bold.rst [hunk 6]
```diff
@@ -87,6 +92,15 @@ reported here.
 
 .. topic:: References
 
+  .. [Patriat2015] Patriat, R., EK Molloy, RM Birn, T. Guitchev, and A. Popov. “Using Edge Voxel Information to 
+     Improve Motion Regression for Rs-FMRI Connectivity Studies.” Brain Connectivity 5, no. 9 (28 2015): 582–95. 
+     doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`_.
```
Comment (oesteban) [inline docs/source/reports/bold.rst:97]:
```suggestion
     doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`__.
```

Comment (oesteban) [reply docs/source/reports/bold.rst:97]: Two underscores is the right choice here.

File: docs/source/reports/bold.rst [hunk 7]
```diff
@@ -87,6 +92,15 @@ reported here.
...
+
   .. [Power2016] Power JD, A simple but useful way to assess fMRI scan qualities.
      NeuroImage. 2016. doi: `10.1016/j.neuroimage.2016.08.009 <http://doi.org/10.1016/j.neuroimage.2016.08.009>`_
 
+  .. [Provins2022] Provins, Céline, Christopher J. Markiewicz, Rastko Ciric, Mathias Goncalves, César Caballero-Gaudes, 
+     Russell Poldrack, Patric Hagmann, and Oscar Esteban. “Quality Control and Nuisance Regression of FMRI, Looking out 
+     Where Signal Should Not Be Found.” OSF Preprints, January 21, 2022. doi: `10.31219/osf.io/hz52v <https://doi.org/10.31219/osf.io/hz52v>`_
```
Comment (oesteban) [inline docs/source/reports/bold.rst:104]:
```suggestion
     Where Signal Should Not Be Found.” OSF Preprints, January 21, 2022.
     doi: `10.31219/osf.io/hz52v <https://doi.org/10.31219/osf.io/hz52v>`__.
```

**Review by oesteban — COMMENTED** [review]
Left a few suggestions.

File: docs/source/reports/bold.rst (hunk 6 above)
Comment (celprov) [reply docs/source/reports/bold.rst:97]: @oesteban what do the one underscore or two underscores do ?

Comment (oesteban) [reply docs/source/reports/bold.rst:97]: LMGTFY...

https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

File: docs/source/reports/bold.rst [hunk 8]
```diff
@@ -41,7 +41,13 @@ The section with visual reports contains:
    signal intensity at the extremes for the identification
    of spikes, the outliers metric, the DVARS and the
    :abbr:`FD (framewise displacement)`. Finally the
-   so-called carpetplot [Power2016]_
+   so-called carpetplot [Power2016]_. 
+   The carpet plot rows correspond to voxelwise time series,
+   and are separated into regions: cortical gray-matter, deep
```
Comment (celprov) [inline docs/source/reports/bold.rst:46]:
```suggestion
   and are separated into regions: cortical gray matter, deep 
```

File: docs/source/reports/bold.rst [hunk 9]
```diff
@@ -41,7 +41,13 @@ The section with visual reports contains:
    signal intensity at the extremes for the identification
    of spikes, the outliers metric, the DVARS and the
    :abbr:`FD (framewise displacement)`. Finally the
-   so-called carpetplot [Power2016]_
+   so-called carpetplot [Power2016]_. 
+   The carpet plot rows correspond to voxelwise time series,
+   and are separated into regions: cortical gray-matter, deep 
+   gray-matter, white matter and cerebrospinal fluid, cerebellum
```
Comment (celprov) [inline docs/source/reports/bold.rst:47]:
```suggestion
   gray matter, white matter and cerebrospinal fluid, cerebellum 
```

**Review by celprov — COMMENTED** [review]
Remove hyphens in gray-matter

**Pull request merged**